test/e2e/run

```diff
@@ -79,6 +79,25 @@ def add_jobs(litani, run_dir, mod):
             litani, "add-job", *job.get("args", []), **job.get("kwargs", {}))
 
 
+def transform_jobs(litani, run_dir, mod):
```

Comment:
Confused by the asynchrony here.  Don't follow what the end-to-end stuff is doing.

Reply:
Resolved during discussion with Kareem.